src/release-specification.ts

```diff
@@ -317,6 +318,55 @@ export async function validateReleaseSpecification(
         }
       }
 
+      // Check to compel users to release packages with breaking changes alongside their dependents
+      if (
+        versionSpecifierOrDirective === 'major' ||
+        (isValidSemver(versionSpecifierOrDirective) &&
+          diff(pkg.validatedManifest.version, versionSpecifierOrDirective) ===
+            'major')
+      ) {
+        const dependentNames = Object.keys(project.workspacePackages).filter(
+          (possibleDependentName) => {
+            const possibleDependent =
+              project.workspacePackages[possibleDependentName];
+            const { dependencies, peerDependencies } =
+              possibleDependent.validatedManifest;
+            return (
+              hasProperty(dependencies, packageName) ||
+              hasProperty(peerDependencies, packageName)
+            );
+          },
+        );
+        const missingDependentNames = dependentNames.filter((dependentName) => {
+          return !unvalidatedReleaseSpecification.packages[dependentName];
+        });
+
+        if (missingDependentNames.length > 0) {
+          errors.push({
+            message: [
+              `The following packages, which depend on released package '${packageName}', are missing from the release spec.`,
```

Comment:
Wondering if we should clarify that it's the major bump that's causing this error:

``` suggestion
              `The following dependents of package '${packageName}', which is being released with a major version bump, are missing from the release spec.`,
```